ui/src/components/sortable/QasSortable.vue

```diff
@@ -1,5 +1,8 @@
 <template>
-  <component :is="tag" ref="sortableItems">
+  <component
+    :is="tag"
+    ref="sortableItems"
+  >
```

Comment:
no asteroid não utilizamos esse padrão

Reply:
q padrão?

Reply:
O correto é `<component :is="tag" ref="sortableItems">`